docs/05/d.md

```diff
@@ -34,7 +34,7 @@ Here's [the typeclass contract for `MonadFilter`][MonadFilterSource]:
 }
 ```
 
-This enables a derivative function/operator `filter` and `filterM`:
+This enables the derivative functions/operators `filter` and `filterM`:
```

Comment:
In the context of Cats, "function" means `MonadFiler[F].filter(fa)(f)`, whereas "operator" is more OO style `fa filter f`, which actually are two different entities, but materialized out of a single line in code thanks to the magic of simulacrum.